### ISSUE TITLE
Send logs to journal

### ISSUE THREAD

**Issue by Trolldemorted** [TheHive-Project/TheHive#1105]
### Request Type
Right now, logs are saved to `/var/log/thehive`, and `journalctl -u thehive` shows nothing except systemd's start and stop messages.

Since thehive emits all logs to stdout just fine (?), removing the unit's `StandardOutput`and `StandardError` directives should be all you have to do ([journal is default](https://www.freedesktop.org/software/systemd/man/systemd.exec.html)).

**Comment by jakubgs** [comment]
Agreed, not sure why the `stdout` and `stderr` is suppressed in the default service definition. Makes no sense to me.
